tests/testthat/test-ev.R

```diff
@@ -260,3 +260,15 @@ test_that("until  issue-513", {
   expect_error(ev(amt=100,addl=5,until=100), "input can include either")
   expect_silent(mutate(ev(amt=100,ii=2,until=168),until=NULL,addl=5))
 })
+
+test_that("until with non-zero dose time gh-1144", {
+  # Every 4 weeks to 16 weeks; four total doses
+  e <- ev(amt = 100, ii = 4*7, until = 16*7)
+  expect_equal(e$addl, 3)
+  # Every 4 weeks to just under 16 weeks; three total doses
+  e <- ev(amt = 100, ii = 4*7, until = 16*7-0.01)
+  expect_equal(e$addl, 2)
```

Comment:
This assertion [fails][1]:

```
── 1. Failure ('test-ev.R:270:3'): until with non-zero dose time gh-1144 ───────
e$addl not equal to 2.
1/1 mismatches
[1] 3 - 2 == 1
```

Doesn't the actual value of 3 (four total doses) match the behavior in the PR description?

[1]: https://github-drone.metrumrg.com/metrumresearchgroup/mrgsolve/1706/1/3

Reply:
Thanks @kyleam ; I thought that was passing but needs another look ... probably got mixed up when I realized the previous behavior wasn't what I as expecting this time around. I'll get it squared away and explain in the PR.